src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -7613,6 +7613,17 @@ components:
         lastSynced:
           description: Last time TDR queried DUOS for its dataset's authorized users
           type: string
+    FirecloudGroupModel:
+      type: object
+      description: >
+        A Firecloud managed group for the authorized users of a resource.
+      properties:
+        groupName:
+          description: Name of Firecloud managed group
+          type: string
+        groupEmail:
+          description: Email of Firecloud managed group
+          type: string
```

Comment:
I think is great, but more of an open question: What are the best practices around defining models in the open api spec vs. manually defining them in code?  I think we usually always define them here in the open api spec, but I don't think we plan to ever actually reference it anywhere else in the spec (i.e. to be returned with an API response).

Reply:
If we don't need it in an API then it shouldn't be in the openapi spec, IMO. There are many downsides to using openapi generated models and it'd be better to avoid them when possible.